charts/piraeus/values.yaml

```
@@ -97,3 +97,12 @@ podDisruptionBudget:
   enabled: true
   minAvailable: 1
   # maxUnavailable: 1
+
+imageConfigOverride: []
+  # - base: example.com/piraeus
+  #   components:
+  #     linstor-csi
```

```suggestion
  #     linstor-csi:
```

To make the yaml valid, right?